src/subsites/SubsiteSecurityReport.php

```diff
@@ -1,4 +1,7 @@
 <?php
+namespace SilverStripe\SecurityReport;
```

Comment:
Tests seem to ignore return or mark skipped. So leaving it with comment for now.

The report won't do anything with the YML commented out, so shouldn't be to big an issue?